### PR TITLE
[ci] Each PR need to pass ci "Dead Link Checker"

### PR DESCRIPTION
**Pull request by zhongjiajie**

### CODE REVIEW

File: .asf.yaml [hunk 1]
```diff
@@ -38,7 +38,8 @@ github:
   protected_branches:
     master:
       required_status_checks:
-        strict: true
+        contexts:
+          - Dead Link Checker
```
Comment (kezhenxu94) [inline .asf.yaml:42]: This might be `CheckDeadLinks`.

```suggestion
          - CheckDeadLinks
```

Comment (zhongjiajie) [reply .asf.yaml]: So it uses job's name?

**Review by kezhenxu94 — APPROVED** [review]

File: .asf.yaml (hunk 1 above)
Comment (kezhenxu94) [reply .asf.yaml]: It's called context name in GitHub, to get this exact name, open a merged PR like https://github.com/apache/dolphinscheduler-website/pull/674, click the "view details" and copy that name

Screen Shot 2022-02-09 at 21 10 12

Comment (zhongjiajie) [reply .asf.yaml]: Thanks for clarification 👍

**Pull request merged**